### ISSUE TITLE
ENH: Add more solar irradiance proxies

### ISSUE THREAD

**Issue by jklenzing** [pysat/pysatModels#124]
# Description
Numerous solar proxies are available through https://lasp.colorado.edu/lisird/.  Current, pysatSpaceWeather accesses some of the F10.7 products here.  These routines could be generalized to support other indices such as Mg II core-to-wing, FISM daily bins, etc.

- [ ] [FISM2 Daily Averages](https://lasp.colorado.edu/lisird/data/fism_daily_hr) -- xarray
- [ ] [FISM2 Flare](https://lasp.colorado.edu/lisird/data/fism_flare_hr) -- xarray
- [ ] [FISM2 Daily Stan Bands](https://lasp.colorado.edu/lisird/data/fism_daily_bands) -- xarray
- [ ] [FISM2 Flare Stan Bands](https://lasp.colorado.edu/lisird/data/fism_flare_bands) -- xarray
- [x] [SORCE Magnesium II Core-to-Wing Index](https://lasp.colorado.edu/lisird/data/sorce_mg_index) -- pandas

## Potential impact

- Is the feature related to an existing problem?
  - F10.7 does not capture short-term variability of solar inputs into the ionosphere
- How critical is this feature to your workflow?
  -  This would enable one-stop access of multiple solar irradiance proxies
- How wide of an impact to you anticipate this enhancement having?
  - Potential use in driving ITM models with multiple inputs
- Would this break any existing functionality?
  - This would be adding an instrument.  Generalizing the routines may have impact depending on implementation
  
## Potential solution(s)
- Additional instruments for lisird, with methods file for access methods

# Alternatives
A clear description of any alternative solutions or features you've considered.
- The exact instrument structure requires discussion, as the existing F10.7 will probably remain as is

# Additional context
Add any other context or screenshots about the feature request here, potentially
including your operational configuration.

**Comment by aburrell** [comment]
There are currently about 112 data sets available.  Which ones do you want most?  If you make a check list, this could be more easily addressed.  A general LISIRD instrument would also belong on the checklist, though given the Madrigal example that's not terribly simple.

**Comment by jklenzing** [comment]
FISM is probably the top priority for current projects.

**Comment by aburrell** [comment]
There are many FISM options, make a checklist with the ones you want and I'll try to get at least one in for the next release.

**Comment by jklenzing** [comment]
checklist added to top-level issue

**Comment by aburrell** [comment]
Thought: FISM2, as a model, may be more appropriately added to pysatModels.

**Comment by jklenzing** [comment]
> Thought: FISM2, as a model, may be more appropriately added to pysatModels.

I like this idea, but note that it would likely make pysatSpaceWeather a dependency there since we use the lisird interface

**Comment by aburrell** [comment]
Not necessarily, we could instead add the lisird interface to each module individually, format it for an individual instrument (as is currently done), add it to pysat, or create a pysatLISIRD interface.  So many possibilities! 🦑

**Comment by jklenzing** [comment]
I think all four FISM products could be a single module.  Proposed properties:
```
platform = 'lasp'
name = 'fism2'
tags = {'daily': 'FISM2 daily averages',
        '60sec': 'FISM2 flare output, 60 second cadence.'}
inst_ids = {'': list(tags.keys()),
            'bands': list(tags.keys())}
```
Potentially could use 'hr' instead of '', which matches the syntax on the site.  Any preference @aburrell?

**Comment by jklenzing** [comment]
Experimenting with using the pysatNASA download routines here, since https is available.

**Comment by aburrell** [comment]
> Potentially could use 'hr' instead of '', which matches the syntax on the site. 

I would prefer that.  As we add more and more data, it becomes increasingly unclear to have empty tags when there are other options available.

**Comment by jklenzing** [comment]
OK.  My current working is that tags define time resolution (daily or flare), with inst_id for wavelength resolution (full spectra vs bands from Solomon and Qian, 2005).

Just noticed the 60sec cadence is not consistent, this is 300sec for bands.  Could change this tag to 'flare' if that would make more sense.
